packages/terra-form-select/src/_OptGroup.jsx

```diff
@@ -24,15 +25,21 @@ const defaultProps = {
   disabled: false,
 };
 
-const OptGroup = ({ children, disabled, label }) => (
+const OptGroup = ({
+  children, disabled, label,
+}) => (
   <li className={cx('opt-group', { 'is-disabled': disabled })} role="group">
     <div className={cx('label')}>
       {label}
     </div>
-    <ul className={cx('options')} role="listbox">
-      {React.Children.map(children, child => (
-        React.cloneElement(child, { disabled: disabled || !!child.props.disabled })))}
-    </ul>
+    <FormattedMessage id="Terra.form.select.option">
+      {option => (
+        <ul className={cx('options')} role="listbox" aria-label={option}>
+          {React.Children.map(children, child => (
+            React.cloneElement(child, { disabled: disabled || !!child.props.disabled })))}
```

Comment:
Although not part of your issue, but why is `!!child.props.disabled` double negated?
It could just be `child.props.disabled`.